x-pack/plugins/secops/server/graphql/sources/resolvers.test.ts

```diff
@@ -48,6 +63,8 @@ describe('Test Source Resolvers', () => {
       {} as GraphQLResolveInfo
     );
     expect(mockSourcesAdapter.getAll).toHaveBeenCalled();
-    expect(data).toEqual(mockSourceData);
+    const expectedData = Object.assign({}, mockSourceData);
+    delete expectedData.status;
```

Comment:
I think what you want here is `omit('status', mockSourceData)` rather then delete?